app/components/team/Edit.js

```diff
@@ -45,24 +46,26 @@ class TeamEdit extends React.Component {
     return (
       <div>
         <form onSubmit={this.handleFormSubmit}>
-          <Panel>
-            <Panel.Section>
-              <TeamForm
-                onChange={this.handleFormChange}
-                errors={this.state.errors}
-                name={this.state.name}
-                description={this.state.description}
-                privacy={this.state.privacy}
-                isDefaultTeam={this.state.isDefaultTeam}
-              />
-            </Panel.Section>
-
-            <Panel.Footer>
-              <Button loading={this.state.saving ? "Saving team…" : false} theme="success">Save Team</Button>
-            </Panel.Footer>
-          </Panel>
+          <TeamForm
+            onChange={this.handleFormChange}
+            errors={this.state.errors}
+            name={this.state.name}
+            description={this.state.description}
+            privacy={this.state.privacy}
+            isDefaultTeam={this.state.isDefaultTeam}
+            saving={this.state.saving}
+            button={this.state.saving ? "Saving team…" : "Save Team"}
+          />
         </form>
 
+        <Panel className="mt3">
+          <Panel.Header>REST API Integration</Panel.Header>
+          <Panel.Section>
+            <p>You can use this UUID to reference this team when using the <a href="/docs/rest-api/pipelines#create-a-pipeline" className="blue hover-navy">Create a Pipeline</a> REST API.</p>
```

Comment:
My only change. "Create Pipeline REST API" rather than "Create a Pipeline" (even though I know that the head in the doc says that). I think in this case it just reads better, and we do that elsewhere sometimes too)